docs/en/getting-started/docker/create-certs.yml

```diff
@@ -13,12 +13,15 @@ services:
         fi;
         chown -R 1000:0 /certs
       '
-    user: "0"
     working_dir: /usr/share/elasticsearch
     volumes: 
       - certs:/certs
       - .:/usr/share/elasticsearch/config/certificates
 
 volumes: 
   certs:
-    driver: local  
\ No newline at end of file
+    driver: local  
+
+networks:
+  elastic:
+    driver: bridge
```

Comment:
Used elastic network here to avoid creating two networks during the setup (previously this one would create `es_default`)